### PR TITLE
feat(Tabs): UXD-474 add shouldUnmount prop

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀

In asset inventory, users will be able to switch between 2 tab panels. Both of them contains collapsible card and closed by default. This PR avoids lots of implementation required on the app side to preserve the opening state and avoid re-fetch data.

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) change to `@paprika/tabs`
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Tabs/src/components/Panel/Panel.js [hunk 1]
```diff
@@ -5,25 +5,35 @@ import * as sc from "./Panel.styles";
 
 const propTypes = {
   children: PropTypes.node,
-  /** Controls if the option is selected or not */
 
+  /** Controls if the option is selected or not */
   isSelected: PropTypes.bool,
+
+  /** Should unmount or not when isSelected is false */
+  shouldUnmount: PropTypes.bool,
```
Comment (nahumzs) [inline packages/Tabs/src/components/Panel/Panel.js:13]: what about `preventUnmount` so it's aligned with preventDefault `javascript land`

File: packages/Tabs/src/components/Panel/Panel.js [hunk 2]
```diff
@@ -5,25 +5,35 @@ import * as sc from "./Panel.styles";
...
 
-  if (!isSelected) {
+  if (!isSelected && shouldUnmount) {
     return null;
   }
 
   return (
-    <sc.Panel {...moreProps} role="tabpanel" data-pka-anchor="tabs.panel">
+    <sc.Panel
+      isSelected={isSelected}
```
Comment (nahumzs) [inline packages/Tabs/src/components/Panel/Panel.js:31]: I guess you should add aria-hidden="true" depending on shouldUnmount too? cc @mikrotron

File: packages/Tabs/src/components/Panel/Panel.js (hunk 1 above)
Comment (allison-c) [reply packages/Tabs/src/components/Panel/Panel.js:13]: preventDefault is a function though, I thought we prefer to use `should` `has` for boolean value?

File: packages/Tabs/src/components/Panel/Panel.js (hunk 2 above)
Comment (allison-c) [reply packages/Tabs/src/components/Panel/Panel.js:31]: There're a `display:none` in styles.js, so aria-hidden will be unnecessary

**Review by jamiek-acl — APPROVED** [review]

**Pull request merged**